README.md

```diff
@@ -1 +1,27 @@
-VTEX Search Page Context app
+# VTEX Search Page Context app
+
+This app is the search page context, contains three contexts.
+
+1 - `SearchPageContext`: pass down variables set by user in blocks (SearchPageContext). This way, the children in flex search-result can access it.
+
+2 - `SearchPageStateContext`: holds variable state in search page, like if the search is loading, what mobile layout is selected, etc. (SearchPageStateContext)
+
+3 - `SearchPageDispatchContext`: holds the dispatch function for the search page state context.
+
+## Usage
+
+```
+
+import {
+  useSearchPageState,
+  useSearchPage,
+  useSearchPageStateDispatch,
+} from 'vtex.search-page-context/SearchPageContext'
+
+...
+
+const { pagination, searchQuery } = useSearchPage()
+const { isFetchingMore } = useSearchPageState()
+const dispatch = useSearchPageStateDispatch()
+
+```
```

Comment:
```suggestion
```

## Troubleshooting

You can check if others are passing through similar issues [here](https://github.com/vtex-apps/search-page-context/issues). Also, feel free to [open issues](https://github.com/vtex-apps/search-page-context/issues/new) or contribute with pull requests.

## Contributing

Check it out [how to contribute](https://github.com/vtex-apps/awesome-io#contributing) with this project. 

## Tests

To execute our tests go to `react/` folder and run `yarn test`
```